### PR TITLE
collects aws account information to snowflake

### PR DESCRIPTION
**Pull request by greg-snowflake**
This is not a perfect solution, but it ingests a collection of data (account id, account alias, number of instances, or an error) to snowflake (and since it's in a try/catch block, if the insert fails for any reason it doesn't kill the rest of the ingest process).

I'd do more refactoring of the script itself (e.g. to use the db functions instead of constructing raw sql) but this script will be obsoleted with a connector soon and it doesn't seem like a productive use of time to fix this up more than this.

### CODE REVIEW

File: src/ingestion/ec2_describe_instances.py [hunk 1]
```diff
@@ -114,12 +116,18 @@ def get_data_worker(account):
                 instances.extend(region)
             except Exception as e:
                 print(f"ec2_describe_instances: account: {account[1]} exception: {e}")
+                db.insert(AWS_ACCOUNTS_INFORMATION_TABLE, values=[(datetime.datetime.utcnow(), account[0], account[1], None, json_dumps(e))])
                 return None
         instance_list = [json.dumps({**instance,"AccountId":account[0]}, default=str) for instance in instances]
+        try:
+            db.insert(AWS_ACCOUNTS_INFORMATION_TABLE, values=[(datetime.datetime.utcnow(), account[0], account[1], len(instance_list), None)])
```
Comment (andrey-snowflake) [inline src/ingestion/ec2_describe_instances.py]: add newlines before column 118 pls

File: src/ingestion/ec2_describe_instances.py [hunk 2]
```diff
@@ -114,12 +116,18 @@ def get_data_worker(account):
...
+        except Exception:
+            print('Failed to insert into AWS_ACCOUNT_INFORMATION table.')
         print(f"ec2_describe_instances: account: {account[1]} instances: {len(instance_list)}")
         return instance_list
     except Exception as e:
         print(f"ec2_describe_instances: account: {account[1]} exception: {e}")
+        db.insert(AWS_ACCOUNTS_INFORMATION_TABLE, values=[(datetime.datetime.utcnow(), account[0], account[1], None, json_dumps(e))])
```
Comment (andrey-snowflake) [inline src/ingestion/ec2_describe_instances.py]: formatting how something is inserted (i.e. `json_dumps`) should be something this code does not concern itself with -- can we put it somewhere into db?

File: src/ingestion/ec2_describe_instances.py (hunk 1 above)
Comment (greg-snowflake) [reply src/ingestion/ec2_describe_instances.py]: resolved

File: src/ingestion/ec2_describe_instances.py (hunk 2 above)
Comment (greg-snowflake) [reply src/ingestion/ec2_describe_instances.py]: Seems reasonable. I added Exception to the tuple of things which are jsony in `db.insert()`, so it'll json_dumps them the same way it does when you pass it a dict, list, or tuple.

**Pull request merged**